tests/utils/skipif.php

```diff
@@ -198,6 +198,7 @@ function skip_if_ssl()
 function skip_if_not_ssl()
 {
     is_ssl(URI) or exit('skip URI is not using SSL');
+    realpath(getenv('SSL_DIR')) !== false or exit('skip SSL_DIR environment not set');
```

Comment:
Is this necessary? Not all SSL tests use the certs: https://github.com/mongodb/mongo-php-driver/blob/master/tests/connect/standalone-ssl-no_verify-001.phpt

Reply:
Side note: [`getenv()`](https://www.php.net/getenv) is actually what returns `false`, so I don't think you'd want to call `realpath()` here even if you do keep this line.

Reply:
Ah, I hadn't considered `getenv` returning `false` as well. `realpath` returns false if the path doesn't exist, but per your comment below this might not entirely cover everything. In light of this and your comment below, I'll create a separate `skip_if_no_ssl_dir` function and include that in all tests that actually do something with the certificates to avoid skipping more tests than necessary.